scripts/create-s3-buckets.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -e
+
+# Executable that waits for a port at an address to be open.
+# For further details go to: https://github.com/ufoscout/docker-compose-wait
+/wait
+
+awslocal s3 mb s3://unstable-elife-style-content-adapter-incoming
+awslocal s3 mb s3://unstable-elife-style-content-adapter-expanded
+awslocal s3 cp ./elife-666-vor-r1.zip s3://unstable-elife-style-content-adapter-incoming
```

Comment:
this archive is small enough - for anything more than 1MB https://git-lfs.github.com/ will be welcome (the hard limit is 100 MB per file on Github, but the soft limit for keeping the repository lean is much lower)